docs/schemes/oauth2.md

```diff
@@ -18,7 +18,8 @@ auth: {
       token_type: 'Bearer',
       redirect_uri: undefined,
       client_id: 'SET_ME',
-      token_key:  'access_token'
+      token_key:  'access_token',
+      state: 'UNIQUE_AND_NON_GUESSABLE'
```

Comment:
This should be random per login not documented here

Reply:
My bad
You're right, `state` must keep random and internal.
We could persist our custom values with auth storage in order to recover them on OAuth2 callback